packages/chord-detect/index.ts

```diff
@@ -19,21 +19,48 @@ const namedSet = (notes: string[]) => {
   return (chroma: number) => pcToName[chroma];
 };
 
-export function detect(source: string[]): string[] {
+export function detect(source: string[], assumePerfectFifth: boolean = false): string[] {
   const notes = source.map((n) => note(n).pc).filter((x) => x);
   if (note.length === 0) {
     return [];
   }
 
-  const found: FoundChord[] = findExactMatches(notes, 1);
+  const found: FoundChord[] = findExactMatches(notes, 1, assumePerfectFifth);
```

Comment:
Here is not so critical, since it's a private function, but I think using `options` will make code more easy to read and maintainable.